gctrpc/rpc.proto

```diff
@@ -518,10 +518,10 @@ message GetAuditEventRequest {
 }
 
 message GetAuditEventResponse {
-    repeated audit_event events = 1;
+    repeated AuditEvent events = 1;
 }
 
-message audit_event {
+message AuditEvent {
```

Comment:
this was originally lower case to signal that it was never meant to be used outside of GetAuditEventResponse but since that is not really needed this could be converted to a nested type

```
message GetAuditEventResponse {
    message AuditEvent {
        string type = 1 ;
        string identifier = 2;
        string message = 3;
        string timestamp = 4;
    }
    
    repeated AuditEvent events = 1;
}```

*feel free to ignore this as its out of scope*

Reply:
Right, If we do the nested way, we would have to use `gctrpc.GetAuditEventResponse_AuditEvent{}` to append to gctrpc.GetAuditEventResponse.Events in rpcserver.go. I think gctrpc.AuditEvent{} is much neater so will keep this as is for now (still ends up exported either way).